src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -114,7 +114,7 @@ public function register(Container $app)
         };
 
         $app['security.encoder.digest'] = function ($app) {
```

Comment:
the service name is not correct anymore, I think you should have add two others services 

 * `security.encoder.bcrypt` returning the `BCryptPasswordEncoder`
 * `security.default_encoder` returning `$app['security.encoder.bcrypt']`

and use `security.default_encoder` instead of `security.encoder.digest` where necessary